### ISSUE TITLE
bug(ci): Latest Release & Artifact Not Working On MacOS Ventura Intel

### ISSUE THREAD

**Issue by alsyundawy** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#722]
### Describe the bug

bug(scope): Latest Release & Artifact Not Working On MacOS Ventura Intel

**Comment by Frigyes06** [comment]
What do you mean by not working? Can you post logs please?

**Comment by AnonymousWP** [comment]
Yes you're completely right, I'll add it to the milestone of 1.1.1. This should have been issue even before 1.1.0 though. Do you recall that?

**Comment by Frigyes06** [comment]
So apparently macos-latest got changed to the M1 machines, as opposed to intel:
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

We will need to add another run on macos-13

**Comment by alsyundawy** [comment]
1.0.3 no self update running well on monterey intel

![Uploading image.png…]()

**Comment by supersophie** [comment]
Hi,

What is the plan for Intel macos releases?

**Comment by supersophie** [comment]
As in some people have got macs running on Intel, and it would really great if we could run this.

**Comment by AnonymousWP** [comment]
@alsyundawy @mallardtheduck @supersophie First: sorry that it had to take this long to get fixed.

It should be fixed in these 2 PRs:

https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/791
https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/793

Could you try it out? Here's the link to the newly generated Intel-artifact: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/actions/runs/12746391399/artifacts/2421626322

**Comment by alsyundawy** [comment]
working like charm, tested. Thx for ward work, nice release

<img width="958" alt="image" src="https://github.com/user-attachments/assets/606103dc-c407-4bd3-a580-1eb7dc77e88f" />

**Comment by AnonymousWP** [comment]
> working like charm, tested. Thx for ward work, nice release
> 
> <img alt="image" width="958" src="https://private-user-images.githubusercontent.com/6451849/402722338-606103dc-c407-4bd3-a580-1eb7dc77e88f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4MDg2NzQsIm5iZiI6MTczNjgwODM3NCwicGF0aCI6Ii82NDUxODQ5LzQwMjcyMjMzOC02MDYxMDNkYy1jNDA3LTRiZDMtYTU4MC0xZWI3ZGM3N2U4OGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDExMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMTNUMjI0NjE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2Y1NzMzZTgwN2ZjZWM2ODQxYjVkMjg0ZTUyNmRmOTgxYjAyYmI2ODA5MjRiMDE2YTFlODZmYThhZjMyNGQ1OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.qsReNU6CqoORKnYtP2_Zz_G440975UYJykZvg4cj3BE">

Great to hear, thanks!

**Comment by alsyundawy** [comment]
hai @AnonymousWP can you release arm & intel on release

thx for hard work

**Comment by AnonymousWP** [comment]
We will once we finish 1.1.1: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/milestone/6
